Add rustdoc team processes (#852)
rust-lang/rust-forge: src/rustdoc/meetings.md

Author: Forgot about it but I should add it for sure.

# Meetings

> The rustdoc team hosts a meeting every second monday on each month on the 

Reviewer: ```suggestion
The rustdoc team hosts a meeting every second Monday of each month on the 
```

[t-rustdoc/meetings channel on zulip](https://rust-lang.zulipchat.com/#narrow/channel/393423-t-rustdoc.2Fmeetings)
at 21:00 CET (UTC+1) and CEST on summer (UTC+2).

A new thread is open a month ahead of time to remind people who want to attend about the time
and the agenda.

Anyone is open to participate to the meetings.

> If you want to add items to the agenda, please comment on the zulip thread for the next meeting

Reviewer: (Side note) If we go down the route of discouraging non-members to edit the agenda themselves directly, we should (sooner than later) migrate to a HackMD "team account" setup (disclaimer: I don't how this works) since nobody will be aware of this change even if we make a Zulip PSA (which would be kinda weird anyway).

I think we briefly touched upon that during AllHands '25. Anyways, this can/should be done separately from this PR.

> If you want to add items to the agenda, please comment on the zulip thread for the next meeting
> the item you want to be discussed. A member of the team will then add it to the agenda.

Reviewer: ```suggestion
If you want to add items to the agenda, please comment on the Zulip thread for the next meeting
about the items you want to see discussed. A member of the team will then add it to the agenda.
```

Sounds more natural (disclaimer: I'm L2 obviously)

Author: English, bad.

> about the items you want to see discussed. A member of the team will then add it to the agenda.

Reviewer 2: This policy is a bit ambiguous

1. the wording partially suggests that team members should also follow this process instead of just adding an item directly
2. it doesn't actually make it clear what purpose this indirection serves.  is it filtering out spam? delaying agenda items to future meetings if the current meeting's agenda is already full?

Author: Indeed, gonna clarify it.

# Meetings

The rustdoc team hosts a meeting every second Monday on each month on the 
[t-rustdoc/meetings channel on zulip](https://rust-lang.zulipchat.com/#narrow/channel/393423-t-rustdoc.2Fmeetings)

> at 21:00 CET (UTC+1) and CEST on summer (UTC+2).

Reviewer 3: Seems better to just link to the calendar here since this will probably fall out of date.

Author: The link is just below.
